projects/packages/blocks/src/class-blocks.php

```diff
@@ -99,7 +115,53 @@ function () use ( $feature_name, $method_name ) {
 			}
 		}
 
-		return register_block_type( $slug, $args );
+		return register_block_type( isset( $block_type ) ? $block_type : $slug, $args );
```

Comment:
Just a small nitpick, maybe we could avoid using the ternary operator inside the function call and instead declare some new variable using it there.

Reply:
Sure thing. I'm just curious why: is it a WordPress or A8C best practice?

Reply:
I'm not sure if it is a formal "best practice", but I can see where it is harder to parse while reading what is being returned with the operator in there.